src/appleseed.python/bindblendermeshconverter.cpp

```diff
@@ -64,6 +64,35 @@ namespace
         char flag, transp;
         short mode, tile, unwrap;
     };
+
+    // Blender 2.8 structures
```

Comment:
I can't comment there but on line 44 it would make sense to specify "Blender 2.79".